src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitButtonTests.cs

```diff
@@ -63,7 +63,8 @@ public Task BitButtonDisabledFocusTest(bool isEnabled, ButtonStyle style, bool a
 
             var bitButton = com.Find(".bit-btn");
 
-            Assert.AreEqual(bitButton.HasAttribute("tabindex") ? bitButton.GetAttribute("tabindex").Equals("-1") : expectedResult, expectedResult);
+            if (bitButton.HasAttribute("tabindex"))
+                Assert.AreEqual(bitButton.GetAttribute("tabindex").Equals("-1"), expectedResult);
```

Comment:
this solution is not correct either. there should be 2 assertions in these tests:
The first assertion should be about checking the existence of a "tabindex" attribute (as before).
The second assertion should be about the value of "tabindex" if the value of expectedResult is true.